go-controller/pkg/kubevirt/types.go

```diff
@@ -12,3 +14,19 @@ const (
 	NamespaceExternalIDsKey      = "k8s.ovn.org/namespace"
 	VirtualMachineExternalIDsKey = "k8s.ovn.org/vm"
 )
+
+// DefaultGatewayReconciler implements a pair of Reconcile functions for
+// ipv4 and ipv6 to update VM's default gw config after live migration
+type DefaultGatewayReconciler struct {
```

Comment:
nit: hmm why is this in types.go file? the functions are living in pod.go? can we just co-locate them?
types.go from what I can tell is just constants?